hs-init.hs

```diff
@@ -49,19 +49,19 @@ import           System.Process        (callCommand, readProcess, showCommandFor
 -----------------------
 
 defaultOwner :: Text
-defaultOwner = "vrom911"
+defaultOwner = "{{DEFAULT_OWNER}}"
 
 defaultName :: Text
-defaultName = "Veronika Romashkina"
+defaultName = "{{DEFAULT_NAME}}"
 
 defaultEmail :: Text
-defaultEmail = "vrom911@gmail.com"
+defaultEmail = "{{DEFAULT_EMAIL}}"
 
 defaultLicense :: Text
-defaultLicense = "MIT"
+defaultLicense = "{{DEFAULT_LICENSE}}"
 
 defaultGHC :: Text
-defaultGHC = "8.0.1"
+defaultGHC = "{{DEFAULT_GHC}}"
```

Comment:
Oops! Now I see that it's not such a good idea to ask user write `DEFAULT_LICENSE` and `DEFAULT_GHC`. It can only be chosen from the list of options, but it's hard to maintain in the installation script. My bad!
So no need to change `defaultLicense` and  `defaultGHC`:
 ```haskell
defaultLicense :: Text		
defaultLicense = "MIT"		
  		  
 defaultGHC :: Text		 
 defaultGHC = "8.0.1"
```